docs/src/main/asciidoc/extension-authors-guide.adoc

```diff
@@ -874,6 +874,37 @@ As a CDI based runtime, Quarkus extensions often make CDI beans available as par
 However, Quarkus DI solution does not support CDI Portable Extensions.
 Instead, Quarkus extensions can make use of various link:cdi-reference.html[Build Time Extension Points].
 
+=== Extension Health Check
+
+Health check is provided via the `quarkus-smallrye-health` extension. It provides both liveness and readiness checks capabilities.
+
+When you write an extension, it can be very handy to provides a default health check for your extension to prevent the developer to do it
+by itself.
+
+In order to provide a health check, you should do the following:
+
+- Import the `quarkus-smallrye-health` extension as an **optional** dependency in your runtime module so it will not impact the size of the application if
+health check is not needed.
```

Comment:
Should that be "not needed" or "not included"?

Reply:
done